repo/packages/P/portworx/0/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://raw.githubusercontent.com/portworx/px-dev/master/images/pwx-48.png",
+    "icon-medium": "https://raw.githubusercontent.com/portworx/px-dev/master/images/pwx-96.png",
+    "icon-large": "https://raw.githubusercontent.com/portworx/px-dev/master/images/pwx-256-1.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "portworx-docker": "portworx/px-dev:latest"
```

Comment:
I've changed from "latest" to "1.1.2".   But we have a fairly responsive release cadence --- about once a month, not including critical updates.   Do you want/expect a corresponding stream of PR's?